corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -187,12 +187,13 @@ def get_case_history(case):
 
     changes = defaultdict(dict)
     for form in XFormInstance.objects.get_forms(case.xform_ids, case.domain):
+        name = xmlns_to_name(case.domain, form.xmlns, form.app_id, form_name=form.form_data.get('@name'))
```

Comment:
It might make sense to just pass the whole `form` to `xmlns_to_name` and extract what's needed from it there.

Reply:
Ah, I guess in some instances it's an unwrapped form.  Maybe not then.

Reply:
Yea, this also get's used for things like export configs which only have an `app_id` and `xmlns`